neutron/l3-agent-evacuate.py

```diff
@@ -128,18 +131,18 @@ def evacuate_l3_agent(api, region, from_agent, to_agent, limit):
            ha_false_routers.append(r)
 
     if not len(ha_false_routers):
-        print "Warning: l3 agent was already evacuated"
+        print("Warning: l3 agent was already evacuated")
         sys.exit(1)
     if limit and (len(ha_false_routers) > limit):
         ha_false_routers = ha_false_routers[0:limit]
-    print "Starting ... Moving a router every 10 seconds\n"
+    print ("Starting ... Moving a router every %s seconds\n" % sleep)
     for r in ha_false_routers:
         r_id = {'router_id': r['id']}
-        print "Removing router %s" % r['id']
+        print("Removing router %s" % r['id'])
         neutron.remove_router_from_l3_agent(from_agent, r['id'])
-        print "Adding   router %s" % r['id']
+        print("Adding   router %s" % r['id'])
         neutron.add_router_to_l3_agent(to_agent, r_id)
-        time.sleep(10)
+        time.sleep(float(sleep))
```

Comment:
Looks like someone hasn't set a default for sleep so this will raise an exception

Reply:
Fixed